source/dotnet/global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.0.100-preview6-012264"
+    "version": "3.0.100"
```

Comment:
Thanks. so now all our .NET builds want core or was it just that specific package you'd created? Unsure why the entire pipeline wants to install Core?